modules/agent-install-ipi-install-root-device-hints.adoc

```diff
@@ -13,7 +13,7 @@ The `rootDeviceHints` parameter enables the installer to provision the {op-syste
 |===
 | Subfield | Description
 
-| `deviceName` | A string containing a Linux device name like `/dev/vda`. The hint must match the actual value exactly.
+| `deviceName` | A string containing a Linux device name like `/dev/vda`. It is recommended to use persistent naming like `/dev/disk/by-path/<device_path>`. The hint must match the actual value exactly.
```

Comment:
It's worth noting that _only_ the actual kernel device (e.g. `/dev/sd<X>`) and the by-path link to it (e.g. `/dev/disk/by-path/<device_path>`) are allowed.
This is what we say in the CRD: https://github.com/metal3-io/baremetal-operator/pull/1264/commits/96f147db56a0e6f30ee4c835dea3a994e3854674#diff-0f3e356f6156566888fdbc0e7c0fa713ee054933106059fc29f702f8d1a554f2R55-R57

I agree that it's recommended to use by-path because it's stable across reboots.

Reply:
@zaneb Rephrased it a bit, PTAL and let me know if it looks good